src/spirc.rs

```diff
@@ -170,6 +172,16 @@ impl<D: SpircDelegate> SpircManager<D> {
             protocol::spirc::MessageType::kMessageTypePause => {
                 self.delegate.pause();
             }
+            protocol::spirc::MessageType::kMessageTypeNext => {
+            	self.index = (self.index + 1) % self.tracks.len() as u32;
+                let track = self.tracks[self.index as usize];
```

Comment:
Ok i'll have a look at it when i can. Implementation shouldn't be that much of a hassle, just extending/replacing/reindexing the track list i guess.

Does your Connect speaker run librespot atm? Or just a store built setup?